packages/flutter_tools/templates/app/web/index.html.tmpl

```diff
@@ -41,5 +41,7 @@
     }
   </script>
   <script src="main.dart.js" type="application/javascript"></script>
+  <noscript><center><h1>Please enable javascript to use {{projectName}}</h1></center></noscript>
```

Comment:
`<center>` is invalid HTML, we shouldn't use that.
Also that's not a header, so `<h1>` is inappropriate here. `<p>` would be more correct.
Also we should localize this into the application's language somehow, rather than always using English. That said, that might be tricky... cc @yjbanov @ferhatb